workspaces/adventure-pack/goodies/typescript/Math.lcm/index.ts

```diff
@@ -0,0 +1,21 @@
+import "../Math.gcd";
+
+declare global {
+  interface Math {
+    lcm(a: number, b: number): number;
+  }
+}
+
+Math.lcm = function (a: number, b: number): number {
+  if (a === 0 || b === 0) {
```

Comment:
As a fun aside, this is a reason why arguably gcd(0, 0) should be infinite. Then when we delete by Infinity below, we naturally get 0 for the lcm 😄